packages/kbn-plugin-generator/sao_template/sao.js

```diff
@@ -80,6 +81,7 @@ module.exports = function({ name }) {
     move: {
       gitignore: '.gitignore',
       eslintrc: '.eslintrc',
+      'package_template.json': 'package.json',
```

Comment:
__note:__ for anyone who will read it: iirc this was required as jest was complaining about syntax errors inside of package.json (we use `if/else` there now).